### PR TITLE
add postgres to ci/builder image

### PR DESCRIPTION
**Pull request by eranrund**
### Motivation

Some upcoming work depends on PostgreSQL being available during CI.

### In this PR
* A new docker image (that was already pushed) with the required packages.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -21,50 +21,55 @@ FROM ubuntu:18.04 AS builder-install
...
+
 # Install build tools and dependencies
 #
 RUN apt-get update -q -q && \
  apt-get upgrade --yes --force-yes && \
  apt-get install --yes --force-yes \
-  automake \
-  autoconf \
   alien \
+  autoconf \
```
Comment (eranrund) [inline docker/Dockerfile:33]: Not a lot here was changed, I just sorted everything. The changes are adding `postgresql-10` and `libpq-dev`

**Comment by garbageslam** [comment]
I think you need to change circle-ci also, otherwise lgtm

**Comment by eranrund** [comment]
> I think you need to change circle-ci also, otherwise lgtm

Good catch

**Review by garbageslam — APPROVED** [review]

**Review by joekottke — APPROVED** [review]
Nice work.  Thanks for the cleanup as well.  Can the Jenkins yaml manifests be updated in this PR as well?

**Comment by eranrund** [comment]
> Nice work. Thanks for the cleanup as well. Can the Jenkins yaml manifests be updated in this PR as well?

Where are those? I am not finding any other references of `1_12` in the repo.

**Comment by joekottke** [comment]
@eranrund Apologies.  You got it already. I just glossed over the last item in the PR.

**Pull request merged**